.github/workflows/pnp-input-api-build.yml

```diff
@@ -241,7 +245,7 @@ jobs:
       - name: Setup NuGet sources
         uses: extenda/actions/setup-nuget-sources@v0
         with:
-          config-file: src/nuget.config
+          config-file: ${{ secrets.path-to-nuget-config != '' && secrets.path-to-nuget-config || 'src/nuget.config' }}
```

Comment:
Is this additional check for empty strings needed? I see [here ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value) there is this example: `group: ${{ github.head_ref || github.run_id }}`.

Reply:
Thanks for suggestion, tested and it works